### PR TITLE
Added enthusiasm

### PR DESCRIPTION
**Pull request by vchowdhary**
### What you did
Added enthusiasm

### Tests
None needed

### Unresolved issues
Steven sucks

### CODE REVIEW

File: src/driver.ml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 let () = Utm.name ()
 let () = Parser.name ()
-let () = Printf.printf "Driver\n"
+let () = Printf.printf "Driver!!\n"
```
Comment (chairmanlu) [inline src/driver.ml]: This dum why u do dis.

File: src/driver.ml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 let () = Utm.name ()
 let () = Parser.name ()
-let () = Printf.printf "Driver\n"
```
Comment (chairmanlu) [inline src/driver.ml]: My code is perfect don't delete.

File: src/driver.ml (hunk 1 above)
Comment (chairmanlu) [inline src/driver.ml]:
```suggestion
let () = Printf.printf "Driver\n"
```

Comment (chairmanlu) [reply src/driver.ml]: I am smart u are dum

**Review by chairmanlu — CHANGES_REQUESTED** [review]
210 > 251

File: src/driver.ml (hunk 1 above)
Comment (vchowdhary) [reply src/driver.ml]: you suck

Comment (vchowdhary) [reply src/driver.ml]:
```suggestion
let () = Printf.printf "Driver!\n"
```
I like this better. WDYT?

**Review by chairmanlu — APPROVED** [review]
Ur still dum

**Comment by shiftsayan** [comment]
> 210 > 251

hmmm

**Pull request merged**